src/index.tsx

```diff
@@ -93,13 +81,7 @@ export const renderApp = () => {
 renderApp();
 
 function bootstrap() {
-  const windowConfig = window.jujuDashboardConfig;
-  const config = windowConfig
-    ? {
-        ...windowConfig,
-        authMethod: getAuthMethod(windowConfig),
-      }
-    : null;
+  const config = window.jujuDashboardConfig || null;
```

Comment:
It looks like this would be handled fine if it was `undefined` so I think we can remove `|| null`.